### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by kkokilep** [ziqi-jin/finetune-anything#15]
Hello, 

Thanks for this amazing repo! I was wondering how good of an MIOU you got on the PASCAL validation set. I trained for about 50,000 iterations and got an MIOU of about 4.12. What performance should I be expecting?

**Comment by kkokilep** [comment]
I discovered an issue in my version of the code where the labels being returned by the dataloader were between 0 and 1 instead of categorical values. This caused the loss to bounce between a fixed range of values instead, instead of decreasing. I'm not sure if this is a general problem or how I was setting paths in the config file, but I thought I should mention this.

**Comment by ziqi-jin** [comment]
> Hello,
> 
> Thanks for this amazing repo! I was wondering how good of an MIOU you got on the PASCAL validation set. I trained for about 50,000 iterations and got an MIOU of about 4.12. What performance should I be expecting?

Hi, the MIOU is from 0 to 100, so 4.12 may not  be a good perfomance

**Comment by ziqi-jin** [comment]
> I discovered an issue in my version of the code where the labels being returned by the dataloader were between 0 and 1 instead of categorical values. This caused the loss to bounce between a fixed range of values instead, instead of decreasing. I'm not sure if this is a general problem or how I was setting paths in the config file, but I thought I should mention this.

the 0/1 problem is happened on predictoin label or ground-truth label？

**Comment by kkokilep** [comment]
It is happening on ground truth labels.

**Comment by ziqi-jin** [comment]
> It is happening on ground truth labels.

Oh, Thanks for your test, I will check it
